install.sh

```diff
@@ -165,12 +165,19 @@ BOOTSTRAP_PACKAGES='zstd crew_mvdir ruby git ca_certificates libyaml openssl'
 # Older i686 systems.
 [[ "${ARCH}" == "i686" ]] && BOOTSTRAP_PACKAGES+=' zlibpkg gcc_lib'
 
-if [[ -n "${CHROMEOS_RELEASE_CHROME_MILESTONE}" ]] && (( "${CHROMEOS_RELEASE_CHROME_MILESTONE}" > "112" )); then
-  # Append the correct packages for systems running v113 onwards.
-  BOOTSTRAP_PACKAGES+=' glibc_lib235 zlibpkg gmp'
+if [[ -n "${CHROMEOS_RELEASE_CHROME_MILESTONE}" ]]; then
+  if (( "${CHROMEOS_RELEASE_CHROME_MILESTONE}" > "112" )); then
+    # Recent Arm systems have a cut down system.
+    [[ "${ARCH}" == "armv7l" ]] && BOOTSTRAP_PACKAGES+=' bzip2 ncurses readline pcre2 gcc_lib'
+  fi
 
-  # Recent Arm systems have a cut down system.
-  [[ "${ARCH}" == "armv7l" ]] && BOOTSTRAP_PACKAGES+=' bzip2 ncurses readline pcre2 gcc_lib'
+  if (( "${CHROMEOS_RELEASE_CHROME_MILESTONE}" < "123" )); then
+    # Append the correct packages for systems running M113 onwards.
```

Comment:
`Append the correct packages for systems running M122 and lower.`?

Reply:
Why was this resolved?  Does the comment make any sense?

Reply:
The text was changed to `# Append the correct packages for systems running M122 and lower.`...

Reply:
Sorry, I didn't refresh to see the updates. :blush: